github/repos_collaborators.go

```diff
@@ -110,6 +110,19 @@ func (s *RepositoriesService) AddCollaborator(ctx context.Context, owner, repo,
 	return s.client.Do(ctx, req, nil)
 }
 
+// AddCollaboratorDirectly adds the specified GitHub user as a collaborator to the given repo.
+//
```

Comment:
I think it's okay to add a note saying that all signs indicate that this might be going away, but we shouldn't actually deprecate it. As of this moment, this is actually a part of the stable GitHub API, and sending invitations is a part of a preview API.

But it'd be good to set correct expectations and help users understand why both methods are available So, I agree that adding a note that AddCollaboratorDirectly does so rather than sending an invitation would be nice. Perhaps put it in parenthesis?